allspice/utils/list_components.py

```diff
@@ -54,6 +60,8 @@ def list_components(
         applies to Altium projects.
     :param ref: Optional git ref to check. This can be a commit hash, branch
         name, or tag name. Default is "main", i.e. the main branch.
+    :param combine_multi_part: If True, multi-part components will be combined
+        into a single component. Currently only works for Altium.
```

Comment:
I don't even think this needs to be a parameter. I can't think of a reason someone wouldn't want to combine these. We typically refer to them as multi-symbol components here, as they really are one part, just separated logically to make design organization easier.

Reply:
This is separated out in `list_components` because you might want to check each part separately, such as in the case when you want to see if all parts of a component are placed, where they are placed etc.

Reply:
Ah, I see 👍